01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,21 @@
+num = 0
+while  num < 20
+  num = num + 1
+
+  if num % 3 == 0  && num % 5 == 0 then 
```

Comment:
```suggestion
  if num % 3 == 0  && num % 5 == 0 
```

Ruby の if では then 省略することが多いので不要です。削除しちゃいましょう。